functorch/_src/partitioners.py

```diff
@@ -148,20 +154,27 @@ def default_partition(
     forward_only_graph = _extract_graph_with_inputs_outputs(joint_module.graph, primal_inputs, fwd_outputs)
     forward_node_names = {node.name for node in forward_only_graph.nodes if node.op != 'output'}
     saved_values = []
+
     for node in joint_module.graph.nodes:
         if node.name not in forward_node_names:
             continue
         # Since we can't save tuple of tensor values, we need to flatten out what we're saving
-        if 'tensor_meta' not in node.meta and node.op == 'call_function':
+        if (
+            'tensor_meta' not in node.meta
+            and node.op == 'call_function'
+            and not is_sym_node(node)
+        ):
             users = node.users
             assert all(user.target == operator.getitem for user in users)
             for user in users:
                 saved_values.append(user)
         else:
             saved_values.append(node)
```

Comment:
why not just directly put things into `saved_value` or `saved_sym_nodes` in the initial loop